### ISSUE TITLE
Stompserver not working?

### ISSUE THREAD

**Issue by fastgar** [EVE-KILL/zKillboard#313]
from zkb.log:
[message:protected] => Unable to connect to stomp.zkillboard.com:61613

config.php:
// Stomp
$stompServer = "tcp://stomp.zkillboard.com:61613";
$stompUser = "guest";
$stompPassword = "guest";

Error was also reported here
https://forums.eveonline.com/default.aspx?g=posts&m=5555427#post5555427

The data i used are from here:
https://forums.eveonline.com/default.aspx?g=posts&m=2994862#post2994862
also notable the posted github repo doesn't exist anymore so i used the php5-stomp packe from the ubuntu repo.

**Comment by sn0opy** [comment]
This repo belongs to Karbos beta.eve-kill.net and thus the correct address is `tcp://eve-kill.net:61613`.

`stomp.zkillboard.com` belongs to https://github.com/3zLabs/zKillboard. File a report there, if stomp isn't working.

**Comment by cvweiss** [comment]
Right, the stomp service is ran by eve-kill.  Also, I believe @karbowiak removed the guest account on Stomp and replaced it with a different stream service. Tagged him to help pipe in on the subject here.

**Comment by karbowiak** [comment]
Stomp is inaccessible for users without an account.

Everyone else should refer to the websocket stream, there is a cli script for it, cli_wsreceive.php and it's also talked about in the README :)

But the entire help, readme, wiki etc. is bound for a rewrite to make everything even more clear.
I just have to find time for it all, so much work to do, just one person /o\

**Comment by fastgar** [comment]
"Websocket is used to stream kills from zKillboard.com and EVE-KILL.net to your killboard. Currently it streams all kills, without any limit.

Websocket server: ws://ws.eve-kill.net/kills/ and wss://ws.eve-kill.net/kills/"

Oh yeah now i know everything :D 
Is the tracker using stomp or websocket?

**Comment by karbowiak** [comment]
tracker used the stomp server, but it's removed, so i have to recode a few things.

And the websocket just fetches every killmail that is spit out, so you'll get a lot of killmails you probably don't want.

It's all a bit iffy at the moment :)
